srcpkgs/autofs/template

```diff
@@ -14,6 +14,7 @@ license="GPL-2.0-or-later"
 homepage="https://www.kernel.org/pub/linux/daemons/autofs"
 distfiles="${KERNEL_SITE}/daemons/autofs/v5/autofs-${version}.tar.xz"
 checksum=b33d1059855664b20eeda26f3e28ff518fb0c3d58f565570af2ae569dc73c0fd
+patch_args="-Nlp1"
```

Comment:
Could we instead fix whitespace in the patch?

Reply:
done